src/apps/run/main.js

```diff
@@ -1,31 +1,34 @@
 import url from "fast-url-parser";
 
-import Juttle from "../../client-lib";
+import Juttle from "juttle-client-library";
 import { getBundle } from "../../client-lib/utils/api";
 
-import '../sass/main.scss'
+import 'juttle-client-library/dist/juttle-client-library.css';
+import '../sass/main.scss';
 
-let currentBundle;
-
-let currentJuttle = new Juttle({
-    serverUrl: 'http://localhost:2000',
-    viewsEl: document.getElementById('juttle-view-layout'),
-    inputsEl: document.getElementById('juttle-input-groups')
-});
+// construct client plus views and inputs
+let client = new Juttle("localhost:8080");
```

Comment:
We shouldn't be hard coding to localhost because otherwise outrigger won't work in our outrigger docker container (see https://github.com/juttle/outrigger/pull/36)